apps/mobile/src/hooks/useNotifications.ts

```diff
@@ -55,6 +57,11 @@ const useNotifications = (): NotificationsProps => {
           if (isAppNotificationEnabled) {
             dispatch(toggleAppNotifications(false))
           }
+
+          if (isOnboarding) {
+            router.navigate('/(tabs)')
```

Comment:
I think that we should be using replace instead of navigate. With navigate on devices with a hardware back button you would be able to go bath to the onboarding screeen. Where with replace you won't be.